pmoves/services/agentgym-rl-coordinator/coordinator/training.py

```
@@ -0,0 +1,382 @@
+        config: Dict[str, Any],
+    ) -> None:
+        """Run training job in background (asyncio task).
+
+        Args:
+            run_id: Training run ID
+            run_uuid: Training run UUID
+            config: PPO configuration
+        """
+        try:
+            # Write config to file
+            config_path = f"/tmp/agentgym_{run_id}_config.yaml"
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/agentgym_"

(S108)